src/runtime_src/doc/toc/xrt_ini.rst

```diff
@@ -37,41 +37,52 @@ The following table lists all supported groups, keys, valid key values, and shor
-+-----------------+------------------------------+-------------------------------------------+
++---------------------+------------------------------+-------------------------------------------+
+|  Key                |  Valid Values                |             Description                   |
++=====================+==============================+===========================================+
+| api_checks          |  [true|false]                |Enable or disable OpenCL API checks:       |
+|                     |                              |                                           |
+|                     |                              |     - true: enable                        |
+|                     |                              |     - false: disable                      |
+|                     |                              |                                           |
+|                     |                              |Default: true                              |
++---------------------+------------------------------+-------------------------------------------+
+| runtime_log         |[null|console|syslog|filename]|Specify where the runtime logs are printed:|
+|                     |                              |                                           |
+|                     |                              |     - null: Do not print any logs.        |
+|                     |                              |     - console: Print logs to stdout       |
+|                     |                              |     - syslog: Print logs to Linux syslog  |
+|                     |                              |     - filename: Print logs to the         |
+|                     |                              |       specified file.                     |
+|                     |                              |       Example, runtime_log=my_run.log     |
+|                     |                              |                                           |
+|                     |                              |Default: console                           |
++---------------------+------------------------------+-------------------------------------------+
+| cpu_affinity        | [{N,N,...}]                  |Pin all runtime threads to specified CPUs. |
+|                     |                              |                                           |
+|                     |                              |Example: cpu_affinity = {4,5,6}            |
++---------------------+------------------------------+-------------------------------------------+
+| polling_throttle    | [N]                          |Specify the time interval in microseconds  |
+|                     |                              |that the runtime library polls the device. |
+|                     |                              |                                           |
+|                     |                              |Default: 0                                 |
```

Comment:
This is only applicable for userspace software scheduler which is for internal use only.  Lets remove from documentation.

Reply:
done. Thanks